File: sdk/javascript/scripts/behaviorsets/MaleMocapLocomotion/init-steer-agents.js
var initSteerAgents = {

	setupSteerAgent : function (charName, prefix){
		console.log("Init Steer Agent")
		var steerManager = scene.getSteerManager()
		steerManager.removeSteerAgent(charName)
		var steerAgent = steerManager.createSteerAgent(charName)
		steerAgent.setSteerStateNamePrefix(prefix)
		steerAgent.setSteerType("example")
		//var sbCharacter = scene.getCharacter(charName)
	}
}


	

File: data/behaviorsets/MaleMocapLocomotion/scripts/init-steer-agents.py
print("Init Steer Agent")


def setupSteerAgent(charName, prefix):
    steerManager.removeSteerAgent(charName)
    steerAgent = steerManager.createSteerAgent(charName)
    steerAgent.setSteerStateNamePrefix(prefix)
    steerAgent.setSteerType("example")
    sbCharacter = scene.getCharacter(charName)
